front/src/app/components/forms/commons/AgencySelector.tsx

```diff
@@ -51,7 +67,9 @@ export const AgencySelector = ({
     formState: { errors, touchedFields },
     control,
   } = useFormContext<SupportedFormsDto>();
-  const [agencyKind, setAgencyKind] = useState<AgencyKindForSelector>("all");
+  const [agencyKinds, setAgencyKinds] = useState<AllowedAgencyKindToAdd[]>(
+    keys(agencyKindToLabel),
+  );
```

Comment:
on peut gérer le cas `shouldFilterDelegationPrescriptionAgencyKind` directement ici :
```suggestion
  const allInitialKinds = keys(agencyKindToLabel)
  const [agencyKinds, setAgencyKinds] = useState<AllowedAgencyKindToAdd[]>(
    shouldFilterDelegationPrescriptionAgencyKind
      ? fitForDelegationAgencyKind
      : allInitialKinds
  );
```